examples/rendezvous/src/main.rs

```diff
@@ -33,7 +33,9 @@ use std::time::Duration;
 async fn main() -> Result<(), Box<dyn Error>> {
     env_logger::init();
 
-    let mut swarm = libp2p::SwarmBuilder::with_new_identity()
+    let keypair = libp2p::identity::Keypair::ed25519_from_bytes([0; 32]).unwrap();
```

Comment:
Can we perhaps add a comment here that says something like: "Results in `PeerId` XYZ which is what we hardcode in the other examples"?

Reply:
Yep! Added a comment. :)